### PR TITLE
Initial project structure

### PR DESCRIPTION
**Pull request by rmfitzpatrick**
These changes introduce a basic gradle project and spotless adoption modeled from otel-java and otel-java-instrumentation.

Also adding github issue and PR templates modeled from collector-contrib.

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#
+# https://help.github.com/articles/dealing-with-line-endings/
+#
+# These are explicitly windows files and should use crlf
+*.bat           text eol=crlf
```
Comment (trask) [inline .gitattributes:5]: we found that this gave us the best spotless experience for developers on windows: https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/707/commits/caa335d1108ddc126db87d7558617f7148f6b67e

File: README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# OpenTelemetry Java Contrib
+
+This project is intended to provide helpful libraries and standalone OpenTelemetry-based utilities that don't fit
+the express scope of the [OpenTelemetry Java](https://github.com/open-telemetry/opentelemetry-java) or
+[Java Instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation) projects.  If you need an
+easier way to bring observability to remote JVM-based applications and workflows that isn't easily satisfied by an SDK
+feature or via bytecode instrumentation, this project is hopefully for you.
```
Comment (trask) [inline README.md:7]: opentelemetry-java-instrumentation is not only for bytecode instrumentation
```suggestion
feature or via instrumentation, this project is hopefully for you.
```

File: .gitattributes (hunk 1 above)
Comment (rmfitzpatrick) [reply .gitattributes]: updated

File: README.md (hunk 2 above)
Comment (rmfitzpatrick) [reply README.md]: of course, corrected.

**Pull request merged**